server/chat-plugins/mafia-data.js

```diff
@@ -208,6 +219,15 @@ const roles = {
 		id: `aligner`,
 		alignment: `solo`,
 		memo: [`Aligner: Once a game, you may target someone. You will win if that person wins. Otherwise, you are a Survivor.`],
+	},
+		amnesic_cop: `amnesiac_cop`,
+		amnesic_seer: `amnesiac_cop`,
```

Comment:
I would make `amnesiac` a mod, not its own role.

Also, does amnesiac normally get to pick who receives? Mafiscum says it's usually random.